eng/Version.Details.xml

```diff
@@ -1,23 +1,23 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Dependencies>
   <ToolsetDependencies>
-    <Dependency Name="Microsoft.DotNet.Arcade.Sdk" Version="7.0.0-beta.23060.4">
+    <Dependency Name="Microsoft.DotNet.Arcade.Sdk" Version="7.0.0-beta.22561.2">
       <Uri>https://github.com/dotnet/arcade</Uri>
-      <Sha>ac5977ebf05451c1adcf24a15d16263e4d25fd0c</Sha>
+      <Sha>f36ea231c234560514ede4c2747897a737ced28f</Sha>
     </Dependency>
   </ToolsetDependencies>
   <ProductDependencies>
-    <Dependency Name="Microsoft.EntityFrameworkCore.Design" Version="7.0.3">
+    <Dependency Name="Microsoft.EntityFrameworkCore.Design" Version="7.0.1">
       <Uri>https://github.com/dotnet/efcore</Uri>
-      <Sha>00e218a90498c21ec14bbaf730a8954c03b42a34</Sha>
+      <Sha>09261bb997c0d82e121cd348d2e43f2b37a65848</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.EntityFrameworkCore" Version="7.0.3">
+    <Dependency Name="Microsoft.EntityFrameworkCore" Version="7.0.1">
       <Uri>https://github.com/dotnet/efcore</Uri>
-      <Sha>00e218a90498c21ec14bbaf730a8954c03b42a34</Sha>
+      <Sha>09261bb997c0d82e121cd348d2e43f2b37a65848</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.EntityFrameworkCore.SqlServer" Version="7.0.3">
+    <Dependency Name="Microsoft.EntityFrameworkCore.SqlServer" Version="7.0.1">
```

Comment:
why are all these versions 7.0.1 instead of 7.0.2

Reply:
there was some mixup with darc updates last release, will update these manually to 7.0.2